examples/profile.js

```diff
@@ -25,22 +25,21 @@ app.MainController = function($http, $scope) {
     this['profileData'] = resp.data['profile'];
   }));
 
-  this['point'] = null;
-};
-
-
-/**
- * @param {Object} point The point object. It correspond to an item in the list
- * of points given to the profile.
- */
-app.MainController.prototype.onProfileHover = function(point) {
-  this['point'] = point;
-};
-
-
-/**
- */
-app.MainController.prototype.onProfileOut = function() {
+  var profileOptions = {
```

Comment:
This object does not have all the properties defined in the `ngeox.profile.ProfileOptions` type alias, so it may make sense to cast the right value.